helpers/typescript.ts

```diff
@@ -115,9 +115,37 @@ export const installTSTemplate = async ({
     console.log(
       `\nUsing managed index from LlamaCloud. Ensure the ${yellow("LLAMA_CLOUD_* environment variables are set correctly.")}`,
     );
+    // copy llamacloud pipeline code for documents
+    await copy(
+      "**",
+      path.join(root, relativeEngineDestPath, "llamaindex", "documents"),
+      {
+        parents: true,
+        cwd: path.join(
+          compPath,
+          "llamaindex",
+          "typescript",
+          "documents",
+          "llamacloud",
+        ),
+      },
+    );
   } else {
     console.log("\nUsing vector DB:", vectorDb ?? "none");
   }
+
+  // remove llamacloud folder in documents
+  await fs.rm(
```

Comment:
if we move the llamacloud pipeline to a own components folder (e.g. `components/llamacloud/typescript`), then we don't need to do this, right?

Reply:
agree, I updated this